### DISCUSSION TITLE
Error install

### DISCUSSION THREAD

**Discussion by projetobleroo** [katoid/angular-grid-layout#70] [General]
I need help, I installed grid layout, but I'm getting this error.

./src/app/main/dashboard/dashboardDefault/dashboardDefault.component.ts:7:0-88 - Error: Module not found: Error: Package path ./public-api is not exported from package C:\Desenvolvimento\ProjetoBleroo\FrontEnd\bleroo\node_modules\@katoid\angular-grid-layout (see exports field in C:\Desenvolvimento\ProjetoBleroo\FrontEnd\bleroo\node_modules\@katoid\angular-grid-layout\package.json)

**Comment by llorenspujol** [comment]
Since you opened: https://github.com/katoid/angular-grid-layout/discussions/71 , I understand that you have solved this issue. You can put here how you solved it.
